cmake/Modules/FindGCOV.cmake

```diff
@@ -5,9 +5,9 @@ FIND_PACKAGE_HANDLE_STANDARD_ARGS(gcov DEFAULT_MSG GCOV_EXECUTABLE)
 # only visible in advanced view
 MARK_AS_ADVANCED(GCOV_EXECUTABLE)
 
-if(NOT GCOV_EXECUTABLE)
+if ( NOT GCOV_EXECUTABLE )
  set(GCOV_FOUND FALSE)
- message("gcov not found! You can't perform coverage tests...")
-elseif(NOT GCOV_EXECUTABLE)
+ message(WARNING "gcov not found! You can't perform coverage tests...")
```

Comment:
**#bug** just like [this](https://github.com/libproject/libproject/pull/105/files#r37697330)